<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QScrollArea" name="originalScrollArea">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QWidget" name="originalScrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>398</width>
       <height>298</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="processedScrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="processedScrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>398</width>
       <height>298</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="sliderL">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>129</y>
      <width>22</width>
      <height>291</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-30</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderA">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>130</y>
      <width>22</width>
      <height>291</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-30</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderB">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>130</y>
      <width>22</width>
      <height>291</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-30</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openAct"/>
    <addaction name="separator"/>
    <addaction name="exportLutAct"/>
    <addaction name="exportImageAct"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="zoomInAct"/>
    <addaction name="zoomOutAct"/>
    <addaction name="fullSizeAct"/>
    <addaction name="fitToWindowAct"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="openAct">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="exportLutAct">
   <property name="text">
    <string>Export 3D LUT</string>
   </property>
  </action>
  <action name="exportImageAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Image</string>
   </property>
  </action>
  <action name="zoomInAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="zoomOutAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="fullSizeAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Full Size</string>
   </property>
  </action>
  <action name="fitToWindowAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fit to Window</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
